pyvista/core/filters/data_set.py

```diff
@@ -4154,15 +4154,20 @@ def plot_over_circular_arc_normal(
         if show:  # pragma: no cover
             plt.show()
 
-    def extract_cells(self, ind, progress_bar=False):
+    def extract_cells(self, ind, assume_unique=False, progress_bar=False):
```

Comment:
This is a breaking change since we do not make progress bar a keyword only arg.  I suspect that it will be very low impact.  Most of the filters I have tried do not really have useful progress bars.

Reply:
```suggestion
    def extract_cells(self, ind, progress_bar=False, assume_unique=False):
```

Reply:
You're right, kwargs should really be added at the end of a function signature.